osf-packager/src/test/java/org/dataconservancy/cos/packaging/OsfContentProviderTest.java

```diff
@@ -146,106 +143,23 @@ public void testCreatePackageWithWiki() throws Exception {
             }
         });
 
-        packager.buildPackage(packageGraph, null);
+        // Read package generation parameters from a resource file.
+        InputStream paramStream =
+                OsfContentProvider.class.getResourceAsStream("/PackageGenerationParams.properties");
 
+        IpmPackager packager = new IpmPackager();
+        packager.buildPackage(contentProvider, null, paramStream);
     }
 
     /**
      * Insures that the system property, {@code osf.client.conf}, is honored by Spring when constructing the application context.
      *
-     * @throws Exception
+     * @throws Exception Thrown when the configuration service cannot be found.
      */
     @Test
     public void testIpmManagerConfigurationResolutionWithSpring() throws Exception {
         System.setProperty("osf.client.conf", "classpath:/org/dataconservancy/cos/packaging/TestSpringClientConfiguration-classpath.json");
-        WbConfigurationService configSvc = IpmPackager.cxt.getBean("wbConfigurationSvc", WbConfigurationService.class);
+        WbConfigurationService configSvc = OsfContentProvider.cxt.getBean("wbConfigurationSvc", WbConfigurationService.class);
         assertEquals("test-host", configSvc.getConfiguration().getHost());
     }
-
```

Comment:
Looking at this now, I'm not sure what happened here.  I don't recall removing these tests, but maybe it happened by accident during refactoring.  Which of these tests should be moved to the forthcoming IpmPackagerTest in dcs-packaging-shared?